eng/versioning/external_dependencies.txt

```diff
@@ -0,0 +1,200 @@
+# Format;
+# groupId:artifactId;dependency-version
+com.fasterxml.jackson.core:jackson-annotations;2.10.0
+com.fasterxml.jackson.core:jackson-core;2.10.0
+com.fasterxml.jackson.core:jackson-databind;2.10.0
+com.fasterxml.jackson.dataformat:jackson-dataformat-xml;2.10.0
+com.fasterxml.jackson.datatype:jackson-datatype-jsr310;2.10.0
+com.fasterxml.uuid:java-uuid-generator;3.2.0
+com.github.spotbugs:spotbugs;4.0.0-beta3
+com.github.spotbugs:spotbugs-maven-plugin;3.1.12.2
+com.google.code.gson:gson;2.8.5
+com.google.guava:guava;24.1.1-jre
+com.microsoft.azure:azure-annotations;1.7.0
+com.microsoft.azure:azure-arm-client-runtime;1.6.15
+com.microsoft.azure:azure-client-authentication;1.6.15
+com.microsoft.azure:azure-client-runtime;1.6.15
+com.microsoft.azure:azure-core;0.9.8
+com.microsoft.azure:azure-keyvault-cryptography;1.2.2
+com.microsoft.azure:azure-mgmt-keyvault;1.11.1
+com.microsoft.azure:client-runtime;1.6.15
+com.microsoft.azure:qpid-proton-j-extensions;1.2.1
+com.microsoft.azure:spotbugs-maven-plugin;1.2.1
+com.microsoft.rest:client-runtime;1.6.15
+com.microsoft.rest.v2:client-runtime;2.1.1
+com.squareup.okhttp3:okhttp;4.2.2
+commons-codec:commons-codec;1.13
+io.micrometer:micrometer-core;1.2.0
+io.micrometer:micrometer-registry-azure-monitor;1.2.0
+io.micrometer:micrometer-registry-graphite;1.2.0
+io.netty:netty-buffer;4.1.42.Final
+io.netty:netty-codec-http;4.1.42.Final
+io.netty:netty-handler;4.1.42.Final
+io.netty:netty-handler-proxy;4.1.42.Final
+io.netty:netty-tcnative;2.0.26.Final
+io.netty:netty-tcnative-boringssl-static;2.0.26.Final
+io.opencensus:opencensus-api;0.24.0
+io.opencensus:opencensus-impl;0.24.0
+io.projectreactor.netty:reactor-netty;0.9.0.RELEASE
+io.projectreactor:reactor-core;3.3.0.RELEASE
+io.reactivex:rxjava;1.2.4
+org.apache.httpcomponents:httpclient;4.3.6
+org.apache.logging.log4j:log4j-api;2.11.1
+org.apache.logging.log4j:log4j-core;2.11.1
+org.apache.qpid:proton-j;0.33.2
+org.asynchttpclient:async-http-client;2.10.4
+org.codehaus.groovy:groovy-eclipse-batch;2.5.8-01
+org.codehaus.groovy:groovy-eclipse-compiler;3.4.0-01
+org.powermock:powermock-api-mockito2;2.0.2
+org.powermock:powermock-module-junit4;2.0.2
+org.slf4j:slf4j-api;1.7.28
+org.slf4j:slf4j-log4j12;1.7.0
+org.slf4j:slf4j-simple;1.7.25
+
+## Test dependency versions
+cglib:cglib-nodep;3.2.7
+com.github.tomakehurst:wiremock-standalone;2.24.1
+com.microsoft.azure:adal4j;1.6.4
+com.microsoft.azure:azure-mgmt-graph-rbac;1.3.0
+com.microsoft.azure:azure-mgmt-keyvault;1.11.1
+com.microsoft.azure:azure-mgmt-resources;1.3.0
+test_jar_com.microsoft.azure:azure-mgmt-resources;1.3.1-SNAPSHOT
```

Comment:
This seems like a weird naming convention and prone to people updating the wrong azure-mgmt-resources line.

Reply:
@JonathanGiles this is the only one named like that. As soon as this is checked in there are going to be issues files for every POM version exception including this one. This is the only entry in the file like this and it's using a 2+ year old Jar.